codestodians/food-basket-api
# [Chore: #154309617] Remove snapshots directory

#### What does this PR do?
- Deprecates the snapshots directory

#### Description of Task to be completed?
- Delete the snapshots directory
- Add the snapshots directory to the `.gitignore` list

#### How should this be manually tested?
- checkout into the `ch-remove-snapshots-directory-154309617` branch.
- Run tests locally, you'd find that the snapshots directory created is no longer in the list of files waiting to be staged.

#### Any background context you want to provide?
- Since Travis runs the tests and creates its own snapshots each time, there is no reason to make the snapshots directory available in the repo codebase (it is not used for anything).
- The `test` and `coverage` commands remain the same. Each contributor should run the tests locally and ensure that the tests pass before concluding work on any story.

#### What are the relevant pivotal tracker stories?
- [#154309617](https://www.pivotaltracker.com/story/show/154309617)
